data_collection/gazette/spiders/base/municipioonline.py

```diff
@@ -0,0 +1,48 @@
+import re
+
+import scrapy
+from dateutil.rrule import DAILY, rrule
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class BaseMunicipioOnlineSpider(BaseGazetteSpider):
+    allowed_domains = ["municipioonline.com.br"]
+
+    def start_requests(self):
+        url = f"https://www.municipioonline.com.br/{self.url_uf}/prefeitura/{self.url_city}/cidadao/diariooficial"
+        yield scrapy.Request(url)
+
+    def parse(self, response):
+        for dt in rrule(freq=DAILY, dtstart=self.start_date, until=self.end_date):
+            formdata = {
+                "__EVENTTARGET": "ctl00$body$txtDataEdicao",
+                "ctl00$body$txtDataEdicao": dt.strftime("%d/%m/%Y"),
+            }
+
+            yield scrapy.FormRequest.from_response(
+                response,
+                formdata=formdata,
+                callback=self.parse_page,
+                cb_kwargs={"page_date": dt.date()},
+            )
+
+    def parse_page(self, response, page_date):
+        editions_list = response.xpath("//*[@id='body_ddlEdicao']/option").getall()
```

Comment:
O seletor css deixaria mais enxuto :)
```suggestion
        editions_list = response.response.css("#body_ddlEdicao option").getall()
```